Amazon.QLDB.Driver/result/AsyncResult.cs

```diff
@@ -45,7 +45,7 @@ internal class AsyncResult : IAsyncResult
             Session session,
             string txnId,
             ExecuteStatementResult statementResult,
-            CancellationToken cancellationToken)
+            CancellationToken cancellationToken = default)
```

Comment:
I'm not strongly against this, but wouldn't it have been easier to just pass default into the constructor for the tests?

Reply:
Easier how? That would require more code changes.

Besides, this allows for more flexibility.